docker-compose.yaml

```diff
@@ -0,0 +1,14 @@
+services:
+  barracuda-thruster-ouput-server:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    user: root
+    container_name: barracuda-thruster-ouput-server
+    tty: true
+    network_mode: host
+    environment:
```

Comment:
this is good, we won't need the vendor and product id env variables bc they were for the ngimu sensor specifically. remove these and it should be good!